packages/builder-vite/package.json

```diff
@@ -27,17 +27,15 @@
     "sveltedoc-parser": "^4.2.1"
   },
   "devDependencies": {
+    "@storybook/mdx1-csf": "^0.0.4",
```

Comment:
`@storybook/mdx1-csf` is included as a dependency of `@storybook/addon-docs`.  However, it uses `@storybook/mdx1-csf@^0.0.1`, but we require `^0.0.3` because of https://github.com/storybookjs/mdx1-csf/pull/9.

Reply:
ok. reverted that